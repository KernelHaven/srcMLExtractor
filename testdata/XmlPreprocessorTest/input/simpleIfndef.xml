<unit language="C">
	<cpp:ifndef>#<cpp:directive>ifndef</cpp:directive><name>A</name></cpp:ifndef>
	<empty_stmt>;</empty_stmt>
	<cpp:endif>#<cpp:directive>endif</cpp:directive></cpp:endif>
</unit>
