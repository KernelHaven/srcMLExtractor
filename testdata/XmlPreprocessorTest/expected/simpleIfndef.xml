<unit language="C">
	<cpp:ifndef>#<cpp:directive>ifndef</cpp:directive><name>A</name>
		<empty_stmt>;</empty_stmt>
	</cpp:ifndef>
</unit>
